lib/jsonWebToken.js

```diff
@@ -165,7 +165,7 @@ exports.verify = function(jwtString, PEMKey, options, callback) {
   if (options.allowMultiAudiencesInToken === false && Array.isArray(payload.aud) && payload.aud.length > 1)
     return done(new Error('mulitple audience in token is not allowed'));
   var payload_audience = Array.isArray(payload.aud) ? payload.aud : [payload.aud];
-  if (payload_audience.indexOf(options.audience) == -1)
+  if (payload_audience.indexOf(options.audience) == -1 && payload_audience.indexOf('spn:'+options.audience) == -1)
```

Comment:
sometimes the `aud` is payload is 'spn:clientid', sometimes it is 'clientid'. Here it can cover both cases.